active_storage_validations.gemspec

```diff
@@ -26,5 +26,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rubocop'
   s.add_development_dependency 'sqlite3'
   s.add_development_dependency 'marcel'
+  s.add_development_dependency 'simplecov', '~> 0.21.2'
```

Comment:
Why the version specification here? Won't it work with 0.22.0? I'd rather keep the version open here as the specific versions will be in the .lock-files anyway.

Reply:
@Animesh-Ghosh bird is the word!

Reply:
@gr8bit removed `simplecov` version specification from `active_storage_validations.gemspec`.